pkg/volume/plugins.go

```diff
@@ -272,17 +273,19 @@ type VolumeConfig struct {
 }
 
 // NewSpecFromVolume creates an Spec from an api.Volume
-func NewSpecFromVolume(vs *api.Volume) *Spec {
+func NewSpecFromVolume(vs *api.Volume, namespace string) *Spec {
 	return &Spec{
-		Volume: vs,
+		Volume:    vs,
+		Namespace: namespace,
```

Comment:
It was that latest consensus (https://github.com/kubernetes/kubernetes/pull/26926#issuecomment-224762431) but I can change to whatever is validated by the team ;)